server/server.go

```diff
@@ -194,6 +200,11 @@ func (s *Server) SetSealer(sealer Sealer) {
 	s.sealer = sealer
 }
 
+// RegisterRPC publishes in the server the methods on rcvr.
+func (s *Server) RegisterRPC(rcvr interface{}) error {
+	return s.dispatcher.Register(rcvr)
+}
```

Comment:
See tests/client_test.go:293, tests/common_test:192. RegisterRPC is called where SetGetCert, SetSealer would be. A client calls RPC methods by creating an RPC client with Conn.RPC(), and using "*rpc.Client.Call(...)".